nodejs/lancedb/merge.ts

```diff
@@ -1,13 +1,20 @@
-import { Data, fromDataToBuffer } from "./arrow";
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Hmmm, are we missing SPDX checks?

Reply:
I don't see any license check in our CI jobs. I can copy over the Lance one later today.